great_expectations/core/usage_statistics/package_dependencies.py

```diff
@@ -152,9 +153,6 @@ class GEDependencies:
         "PyHive",
         "thrift",
         "thrift-sasl",
-        # requirements-dev-test-pipeline.txt:
-        "pytest-azurepipelines",
```

Comment:
We no longer use `pytest-azurepipelines`.  `pytest-random-order` has been moved to the contrib requirements so is now in a different section of this list at line 133.

Reply:
if we remove it here, should we remove all instances in our Azure YAML?

Reply:
It only appears in the `mssql` tests which are outside of docker at this time. We run a coverage report in this step using it though I'm not sure the state of that. Since we are moving to a different coverage solution I'd rather we remove it from that step then.